qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature

```diff
@@ -0,0 +1,126 @@
+
+	
+@Type:Positive
+@HappyPass
+@Priority:Critical
+Scenario Outline: 3.1_01 Fill the Send CV form form with correct data
+	
+	When user enters correct data into the field "Name": <Name>
+	And user enters correct data into the field "Email": <Email>
+	And user presses "Attach CV" button and choses a file in correct format (PDF / DOC / DOCX / RTF / Max 10 Mb)
+	And user checks the checkbox "I agree to the personal data processing"
+	And user presses "Send" button
+	Then the following message should be displayed on the screen:
+		| "Thanks for being awesome!" |
+	And "Ok" button should be visible
+	Examples: 
+		|Name       |Email            |
+		|Test Testov|test@quantori.com|
+
+
+@Type:Positive
+@Priority:Critical
+Scenario Outline: 3.1_02 Fill the field "Name" with correct data
+	When user enters "<sample data>" (<comment>) value into the "Name" field
+	And user clicks on another form field to move the focus
+	Then a green check mark appears at the right side of the "Name" field
+	Examples:
+		|sample data|comment                         |
+		|Olga N     |two words, English              |
+		|Ольга Н    |two words, non-English          |
+		|Olga S N   |more than two words, English    |
+		|Ольга С Н  |more than two words, non-English|
+		
+
+
+@Type:Positive
+@Priority:Critical
+Scenario Outline: 3.1_03 Attach correct CV file
+	When user presses "Attach CV" button
+	And user choses a file in <Format> format 
+	And with <Extension> extension 
+	And attached file size is less than or equal to 10 Mb
+	And attached file size is more than 0 bytes
+	Then attached file name should be displayed under the "Attach CV" button
+	Examples:
+		|Format|Extension|
+		|PDF   |.pdf     |
+		|DOC   |.doc     |
+		|DOCX  |.docx    |
+		|RTF   |.rtf     |
+	
+@Type:Negative
+@Priority:Critical
+Scenario Outline: 3.1_04 Fill the field "Name" with incorrect data
+	When user enters "<sample data>" (<comment>) value into the "Name" field
+	And user clicks on another form field to move the focus
+	Then a red exclamation mark (!) appears at the right side of the "Name" field
+	Examples:
+		|sample data|comment              |
+		|           |empty field          |
+		|           |one or more spaces   |
+		|Olga       |one word             |
+		|Ольга      |one word, non-English|
+		|11111553   |digits, one word     |
+		|132342 678 |digits, two words    |
+
+@Type:Negative
+@Priority:Critical
+Scenario Outline: 3.1_05 Fill the field "Email" with incorrect data
+	When user enters "<sample data>" (<comment>) value into the "Email" field
+	And user clicks on another form field to move the focus
+	Then a red exclamation mark (!) appears at the right side of the "Email" field
+	Examples:
+		|sample data |comment                  |
+		|            |empty field              |
+		|            |one or more spaces       |
+		|@.com       |empty username and domain|
+		|вася@mail.ru|with non-English symbols |
```

Comment:
Non-English symbols are allowed, but not in domain

Reply:
will fix